components/brave_wallet/browser/solana_message_unittest.cc

```diff
@@ -309,4 +309,25 @@ TEST(SolanaMessageUnitTest, FromToValue) {
   }
 }
 
+TEST(SolanaMessageUnitTest, GetNumberOfSigners) {
```

Comment:
Add a test case that when we have a transaction which # of signers unexpectedly exceed UINT8_MAX.